CMSgov/cms-carts-seds
# MDCT 190: Refactor Annual State Template to use an s3 bucket

# Description

This swaps the Fiscal Year Template from living locally on the site to having its own dedicated S3 bucket. 

A couple things should be noted:
1. After first PR go through I will remove the MDCT-190 branch condition from serverless file and double check a new branch is created in main

2. Also on deploy, we need to set SSM up so that there’s a new parameter that points to the main/val/prod buckets.We will also need to upload the pdf to main/val/prod buckets

## How to test

1. Make sure you're on the VPN
2. Log into cloudtamer.cms.gov
3. Navigate to Projects -> mdct-carts-dev -> cloud access -> aws account -> mdctcarts-developer-admin -> short-term access keys
4. Copy the Option 1 block of exported AWS keys
5. Paste the block of exported AWK keys into your terminal
6. ./dev local
7. Sign in as a state user (Like stateuser2)
8. Click the download button on the homepage

## Dependencies

Please spell out any dependencies for this change -- ex -- requires an install / update / migration etc

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
